src/main/resources/openapi/PrintServiceAPI.yaml

```diff
@@ -0,0 +1,155 @@
+openapi: 3.0.0
+info:
+  title: Print Service API
+  version: '1.0'
+  description: API Specification EROP and Voter Card API interactions with Print Service
+  contact:
+    name: Krister Bone
+    email: krister.bone@valtech.com
+servers:
+  - url: 'https://api.printprovider.int.erop.ierds.uk'
+components:
+  schemas:
+    PrintRequest:
+      title: PrintRequest
+      type: object
+      description: Represents a request to print a voter certificate. This is the SQS message to be sent to the Print Service from the Voter Card API Service
```

Comment:
I don't understand what this spec if for, and what this model type is?
There are no `paths` elements in this spec, so is it a REST API?
As I understand it VCA etc will request a print via an SQS message. On that basis I'm guessing this `PrintRequest` model is the SQS message? If so it doesn't live in an openAPI spec like this (ref: the similar work @Neil-Massey and I are doing re: notifications-api)

Reply:
WIP - will need paths, just need to question what APIs are actually needed. But this currently is only SQS models.